basic-steps/src/main/resources/org/jenkinsci/plugins/workflow/steps/CoreStep/help.html

```diff
@@ -0,0 +1,9 @@
+<div>
+    <p>
```

Comment:
If you are referring to a padding inside the help box, yes, it is there, but it is outside the control of this plugin, so what's the correct way to go? A `<p>` tag here is the correct HTML markup, should I remove it because there is a CSS behaviour that should fixed somewhere else?